lxd/vm_qemu.go

```diff
@@ -361,19 +361,20 @@ func (vm *vmQemu) mount() (ourMount bool, err error) {
 	return
 }
 
-// unmount unmounts the instance's config volume if needed.
-func (vm *vmQemu) unmount() error {
-	pool, err := vm.getStoragePool()
+// unmount the instance's config volume if needed.
+func (vm *vmQemu) unmount() (unmounted bool, err error) {
```

Comment:
We usually avoid named return arguments as that makes it confusing during review. I'll update the commit to undo that.

Reply:
Or just push one to undo that pattern from vm_qemu.go